packages/client/src/containers/PageNotFound/PageNotFound.Container.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import { Typography } from '@mui/material';
+
+export const PageNotFound = () => {
+  return (
+    <Section className="page-not-found-container">
+      <Typography variant="h2" color="white">
+        Page Not Found 😥
+      </Typography>
+    </Section>
+  );
+};
+
+const Section = styled.section`
+  height: 100vh;
```

Comment:
This will push the footer out of view, but I guess we will deal with it with the overall footer fix 🙏🏽